### PR TITLE
let getComplexData read from XML

### PR DESCRIPTION
**Pull request by JonathanMeans**
https://github.com/ngageoint/six-library/issues/119

@msciaramitaro

### CODE REVIEW

File: six/modules/python/six.sicd/source/six_sicd.i [hunk 1]
```diff
@@ -306,6 +308,19 @@ def readRegion(inputPathname, startRow, numRows, startCol, numCols, schemaPaths
     getWidebandRegion(inputPathname, schemaPaths, complexData, startRow, numRows, startCol, numCols, widebandBuffer)
 
     return widebandData, complexData
+
+def writeXML(complexData, pathname):
```
Comment (asylvest) [inline six/modules/python/six.sicd/source/six_sicd.i]: Typically I'd just think of Swig code going in the .i file but I guess we don't have a good precedent for utility functions like this going elsewhere?  Would it be straightforward to put this in its own .py file somewhere that waf would still install?  If that's a big pain, it's ok to leave it here for now.

**Comment by JonathanMeans** [comment]
I've made the changes. I think I fixed https://github.com/ngageoint/six-library/issues/43 as well.

File: six/modules/python/six.sicd/source/six_sicd.i [hunk 2]
```diff
@@ -306,12 +308,13 @@ def readRegion(inputPathname, startRow, numRows, startCol, numCols, schemaPaths
...
 
     return widebandData, complexData
+
+
 def writeAsNITF(outFile, schemaPaths, complexData, image):
     writeNITF(outFile, schemaPaths, complexData,
         image.__array_interface__["data"][0])
 
 def readFromNITF(pathname, schemaPaths=VectorString()):
-    pathname = pathname + ".nitf"
```
Comment (asylvest) [inline six/modules/python/six.sicd/source/six_sicd.i]: This seemed like a strange line to be in there... I assume that was in there by accident before and you removed it with intention here?

**Comment by asylvest** [comment]
I see that `__init__.py` got touched but GitHub isn't showing any diff for it... did you change permissions on it or something or what actually changed?  How did this (or something else) affect #43?

File: six/modules/python/six.sicd/source/six_sicd.i (hunk 2 above)
Comment (JonathanMeans) [inline six/modules/python/six.sicd/source/six_sicd.i]: There were actually a number of read/write functions that append extensions, so I wouldn't say I put it there by accident. But this is an intentional removal, yes.

**Comment by JonathanMeans** [comment]
All I did was create an empty __init__.py. Then waf should put it in the correct location when it installs. 

Somehow, the __init__.py gets created if you do `waf clean configure [...] install`, but if you just try to remove the install dir then do `waf install`, that's when the error would show up.

**Comment by JonathanMeans** [comment]
The __init__.py is gone from this now.

**Comment by asylvest** [comment]
This is good to go

**Pull request merged**